packages/ui/src/utils/ThemeUtil.ts

```diff
@@ -637,12 +638,15 @@ export const elementStyles = css`
   button:disabled > wui-all-wallets-image,
   button:disabled > wui-network-image,
   button:disabled > wui-image,
-  button:disabled > wui-icon-box,
   button:disabled > wui-transaction-visual,
   button:disabled > wui-logo {
     filter: grayscale(1);
   }
 
+  button:disabled > wui-icon-box {
+    opacity: 0.3;
```

Comment:
This should be `0.5` for the icon, mentioned [here](https://walletconnect.slack.com/archives/C06B51DMENL/p1721739748814159?thread_ts=1721375493.320809&cid=C06B51DMENL)